clog/loggers.py

```diff
@@ -269,7 +269,7 @@ def close(self):
             self.stream_files[name].close()
 
     def _create_file(self, stream):
-        return open(os.path.join(config.log_dir, stream + '.log'), 'ab', 0)
+        return open(os.path.join(config.log_dir.value, stream + '.log'), 'ab', 0)
```

Comment:
`config.log_dir` is always set via `get_string` isn't it? I might be missing something.

Anyway, we should probably just try to get the staticconf PR merged and then bump it here right?